asclepius/asclepius/enrich.py

```diff
@@ -1,8 +1,43 @@
-def enrich(ddis):
-    def _enrich(ddi):
-        result = {'harmful': bool(hash(ddi['uri']) % 2),
-                  'spacing': abs(hash(ddi['uri']))%14 + 1}
-        result.update(ddi)
-        return result
+from hashlib import sha1
+import csv
+import sys
+import dinto
+import clize
+from clize.parameters import one_of
+
+UNITS = ('hr', 'day')
```

Comment:
I know it doesn't make much sense but can we do all of his units, (sec, min, hr, day, week, yr)?